tests/unit/no-baseline.js

```diff
@@ -93,12 +93,9 @@ describe('No baseline', function() {
 
     // create local mugshot instance with different options
     var mugshot = new Mugshot(browser, options);
-    var result = {
-      isEqual: false,
-      baseline: baselinePath
-    };
 
     mugshot.test(noSelector, callback);
-    expect(callback).to.have.been.calledWithExactly(null, result);
+    expect(callback)
+        .to.have.been.calledWithExactly(new Error('Baseline missing'));
```

Comment:
I would use `calledWith` instead because we don't really care about the results object.